README.md

```diff
@@ -140,6 +140,7 @@ Add to your `.babelrc`'s plugins array.
 
 ## Benchmarks
 > Bootstrap: `npm run bootstrap`
+> Build: `npm run build`
```

Comment:
In the travis test output, build is ran after bootstrap, so that's the order I put it in in the readme...?